vcd/datasource_vcd_org_vdc.go

```diff
@@ -177,6 +177,16 @@ func datasourceVcdOrgVdc() *schema.Resource {
 				Computed:    true,
 				Description: "True if discovery of vCenter VMs is enabled for resource pools backing this VDC. If left unspecified, the actual behaviour depends on enablement at the organization level and at the system level.",
 			},
+			"elasticity": &schema.Schema{
+				Type:        schema.TypeString,
```

Comment:
Is there a reason it is `TypeString`? Could `d.GetOkExists` solve the problem?

Reply:
I'll try - seems may solve bool issue and GetOk

Reply:
That worked - Changed.